build/cmake/functions.cmake

```diff
@@ -836,6 +835,7 @@ function(wx_add name group)
     elseif(group STREQUAL Tests)
         target_include_directories(${target_name} PRIVATE ${wxSOURCE_DIR}/tests)
         target_include_directories(${target_name} PRIVATE ${wxSOURCE_DIR}/3rdparty/catch/include)
+        target_include_directories(${target_name} PRIVATE ${wxTOOLKIT_INCLUDE_DIRS})
```

Comment:
Also need other guys to review this: should we only use these folders for build, not for usage?

Reply:
This is for `wxWidgets` own projects, like `test_base` and `test_gui`. It uses `glib` directly so it should also include those headers.